CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished
+* larva-patterns - Render srcset and sizes attributes instead data-srcset and data-sizes for images aren't lazy.
```

Comment:
Let's change the end of the sentence to: "for images **that** aren't lazy loaded."

Reply:
Also, please add another line noting that we've added a flag to skip lazy-loading for images in the initial viewport.

`* larva-patterns - Add flag to skip lazy loading an image that appears in the initial viewport.`